requirements.txt

```diff
@@ -1,32 +1,52 @@
-#
-# Requirements for wger for production
-#
-
-Django>=1.9,<1.10
-django-recaptcha
-reportlab
-django_mobile
-django-formtools>=1.0,<1.1
-bleach
-python-mimeparse
-pillow
-easy-thumbnails
-django_compressor
-icalendar
-django-bootstrap-breadcrumbs
-sphinx
-django-sortedm2m
-django-bower
-invoke
-requests
-whitenoise==3.3.0
+alabaster==0.7.10
```

Comment:
manually add your new requirements rather than running pip freeze, it adds unnecessary items to the requirements. txt